<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="35226" started-at="2018-10-26T08:51:14Z" finished-at="2018-10-26T08:51:49Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="35226" started-at="2018-10-26T08:51:14Z" finished-at="2018-10-26T08:51:49Z">
      <class name="com.crm.qa.testcase.HomePageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.crm.qa.testcase.HomePageTest@598067a5]" name="setUp" is-config="true" duration-ms="22518" started-at="2018-10-26T08:51:14Z" finished-at="2018-10-26T08:51:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyUserTitelTest()[pri:1, instance:com.crm.qa.testcase.HomePageTest@598067a5]" name="verifyUserTitelTest" duration-ms="418" started-at="2018-10-26T08:51:36Z" finished-at="2018-10-26T08:51:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserTitelTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.crm.qa.testcase.HomePageTest@598067a5]" name="tearDown" is-config="true" duration-ms="637" started-at="2018-10-26T08:51:37Z" finished-at="2018-10-26T08:51:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.crm.qa.testcase.HomePageTest -->
      <class name="com.crm.qa.testcase.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.crm.qa.testcase.LoginPageTest@3c0ecd4b]" name="setUp" is-config="true" duration-ms="10956" started-at="2018-10-26T08:51:37Z" finished-at="2018-10-26T08:51:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginPageTitleTest()[pri:1, instance:com.crm.qa.testcase.LoginPageTest@3c0ecd4b]" name="loginPageTitleTest" duration-ms="15" started-at="2018-10-26T08:51:48Z" finished-at="2018-10-26T08:51:48Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [#1 Free CRM for Any Business: Online Customer Relationship Software] but found [#1 Free CRM software in the cloud for sales and service]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [#1 Free CRM for Any Business: Online Customer Relationship Software] but found [#1 Free CRM software in the cloud for sales and service]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.crm.qa.testcase.LoginPageTest.loginPageTitleTest(LoginPageTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.crm.qa.testcase.LoginPageTest@3c0ecd4b]" name="tearDown" is-config="true" duration-ms="639" started-at="2018-10-26T08:51:48Z" finished-at="2018-10-26T08:51:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.crm.qa.testcase.LoginPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
